08_cell_line_prediction/scripts/plot_fs_for_genes.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Script to run plot_univariate_fs_results.ipynb for several different
+# genes using papermill, writing the results to the fs_plots directory.
+
+# Note that this must be run from the parent directory
+# {repo_root}/08_cell_line_prediction.
+
+kernel_name="pc-eval"
+
+papermill_output_dir="./papermill_output_nbs"
+mkdir -p ${papermill_output_dir}
+
+genes=(
+  "TP53"
+  "EGFR"
+  "PIK3CA"
+  "PTEN"
+  "RB1"
+  "KRAS"
+  "BRAF"
+)
+
+for gene in "${genes[@]}"; do
+
+    cmd="papermill plot_mutation_prediction_results.ipynb "
+    cmd+="${papermill_output_dir}/plot_mutation_prediction_results.run.ipynb "
+    cmd+="-k ${kernel_name} "
+    cmd+="-p gene ${gene}"
+    echo "Running: $cmd"
+    eval $cmd
```

Comment:
I hadn't heard of papermill before, I assume it's better for running notebooks than `nbconvert --execute`?

Reply:
It's pretty much the same but it takes command line parameters, and `nbconvert --execute` doesn't support them AFAIK. I think officially Jupyter notebooks aren't really intended to take parameters like a function/Python module would, but I find it convenient if I want to run the same notebook for multiple genes or multiple cancer types, etc.